rundeckapp/grails-app/services/rundeck/services/JobReferenceImpl.groovy

```diff
@@ -27,19 +27,25 @@ class JobReferenceImpl implements JobReference {
     String jobName
     String groupPath
     String serverUUID
+    Boolean multipleExecutions
+    String maxMultipleExecutions
```

Comment:
why a String?

Reply:
TBH because it was a string in the domain object. I will better refactor that part a bit.